Kitodo/src/main/java/org/kitodo/production/dto/ProjectDTO.java

```diff
@@ -223,12 +224,12 @@ public void setClient(ClientDTO client) {
     }
 
     /**
-     * Get list of templates.
+     * Get list of active templates.
      *
-     * @return list of templates as TemplateDTO
+     * @return list of active templates as TemplateDTO
      */
     public List<TemplateDTO> getTemplates() {
-        return templates;
+        return templates.stream().filter(TemplateDTO::isActive).collect(Collectors.toList());
```

Comment:
I think it would be better not to add logic to a DTO bean. If inactive templates are not needed when querying a project for its templates, they should probably not be indexed in the first place?

Reply:
I agree with you in principle about avoiding business logic in DTOs, but I think we still need to add inactive templates to the index. Most list views are filled from the index and we wouldn't be able to edit and re-activate inactive templates if they were _not_ in the index and consequently would not appear in the list, would we?